src/lib/Zuora.js

```diff
@@ -0,0 +1,124 @@
+import type {config} from './config'
+import request from 'request'
+import NamedError from './NamedError'
+
+export type query = {
+  'name':string,
+  'query':string
+}
+
+export class Zuora {
+  constructor (config:config) {
+    this.authorization = {'Authorization': 'Basic ' + Buffer.from(`${config.username}:${config.password}`).toString('base64')}
+  }
+  async query (name:string, ...queries:query) {
+    const exportQueries = queries.map((q) => { return {...q, 'type': 'zoqlexport'} })
+    const options = {
+      method: 'POST',
+
+      uri: `${config.url}/apps/api/batch-query/`,
+      json: true,
+      body: {
+        'format': 'csv',
+        'version': '1.0',
+        'name': 'Fulfilment-Queries',
```

Comment:
can't we use the query name in the query type for this?
I think we need these names to identify the queries later